### ISSUE TITLE
[BUG] Git-Hooks not working

### ISSUE THREAD

**Issue by dertuerke** [mtlynch/picoshare#135]
Hi @mtlynch,
the git hook script are not working :S

`./dev-scripts/enable-git-hooks`

```
+ set -e
+ set -u
+++ dirname ./dev-scripts/enable-git-hooks
++ cd ./dev-scripts
++ pwd
+ readonly SCRIPT_DIR=/mypath/GitHub/picoshare/dev-scripts
+ SCRIPT_DIR=/mypath/GitHub/picoshare/dev-scripts
+ cd /mypath/GitHub/picoshare/dev-scripts/..
+ [[ -L .git/hooks ]]
+ [[ -d .git/hooks ]]
+ ln --symbolic --force ../dev-scripts/git-hooks .git/hooks
ln: illegal option -- -
usage: ln [-s [-F] | -L | -P] [-f | -i] [-hnv] source_file [target_file]
       ln [-s [-F] | -L | -P] [-f | -i] [-hnv] source_file ... target_dir
       link source_file target_file
```

**Comment by dertuerke** [comment]
Found it. `--symbolic` & `--force` dosent exists on Mac-Bash

**Comment by dertuerke** [comment]
#137 fixed here

**Comment by mtlynch** [comment]
Fixed in #137
